source/libraries/CompileSolidity.ts

```diff
@@ -24,12 +25,12 @@ class SolidityContractCompiler {
     public async compileContracts(): Promise<CompileContractsOutput> {
         try {
             // Compile all contracts in the specified input directory
-            const inputJson: CompilerInput = await this.generateCompilerInput();
-            const compilerOutput: any = compileStandardWrapper(JSON.stringify(inputJson), this.readCallback);
-            const compileOutputJson = JSON.parse(compilerOutput);
-            if (compileOutputJson.errors) {
+            const compilerInputJson: CompilerInput = await this.generateCompilerInput();
```

Comment:
No need for the type here, it should be inferred from the return value of `generateCompilerInput`.